cmd/siot/main.go

```diff
@@ -14,15 +18,56 @@ import (
 	"github.com/simpleiot/simpleiot/sim"
 )
 
+func send(portal, sample string) error {
+	frags := strings.Split(sample, ":")
+	if len(frags) != 4 {
+		return errors.New("format for sample is: 'devId:sensId:value:type'")
+	}
+
+	deviceID := frags[0]
+	sampleID := frags[1]
+	value, err := strconv.ParseFloat(frags[2], 64)
+	if err != nil {
+		return err
+	}
+
+	sampleType := frags[3]
+
+	sendSamples := api.NewSendSamples(portal, deviceID, time.Second*10, false)
+
+	err = sendSamples([]data.Sample{
+		data.Sample{
```

Comment:
No need to use a `data.Sample` literal here; Go knows what type of elements the slice literal contains. This is valid syntax:

```
[]T{
  {
  },
}
```

Reply:
fixed.